dist_tables/dml.rst

```diff
@@ -38,6 +38,70 @@ For example:
 
     If COPY fails to open a connection for a shard placement then it behaves in the same way as INSERT, namely to mark the placement(s) as inactive unless there are no more active placements. If any other failure occurs after connecting, the transaction is rolled back and thus no metadata changes are made.
 
+INSERT INTO ... SELECT
```

Comment:
@begriffs and I chatted about this and I'm summarizing our chat below.

We decided to refer to another section for the colocation image. We also decided to remove the UPSERT image.

We also came up with the following outline.

1. Talk in general about data roll-up and aggregation -- This is similar to our first and second paragraphs without mention of Citus
2. Talk about data roll-up in distributed systems and segue way into Citus
3. Create raw events table ``page_views`` and distribute that table by site_id
4. Show example SELECT query and mention that this query will need to go over the entire results each time
5. Create table ``daily_page_views`` and distribute by site_id 
6. INSERT INTO SELECT distributed table and corresponding paragraphs
7. Run example SELECT query on the aggregated table
8. Other paragraphs
9. Cut last two paragraphs because we're already addressing them in 6.1